.gitsubprojects

```diff
@@ -1,5 +1,5 @@
 # -*- mode: cmake -*-
-git_subproject(Brion https://github.com/BlueBrain/Brion.git 015515f)
+git_subproject(Lunchbox https://github.com/Eyescale/Lunchbox.git 87a9ac3)
+git_subproject(Brion https://github.com/BlueBrain/Brion.git d1f9c43)
```

Comment:
Hmm, I always keep wondering if we should have Lunchbox in here (and in all similar cases). Effectively this forces a different Lunchbox onto Brion then with a separate compile, creating potential problems?